oep-template.rst

```diff
@@ -4,42 +4,46 @@ OEP-XXX: OEP Template
-+---------------+-------------------------------------------+
++-----------------+--------------------------------------------------------+
+| OEP             | :doc:`OEP-XXXX-YYYY-ZZZZ </oeps/oep-XXXX-YYYY-ZZZZ>`   |
+|                 |                                                        |
+|                 | * <XXXX is the next available OEP number>              |
+|                 | * <YYYY is the abbreviated Type: proc | bp | arch>     |
+|                 | * <ZZZZ is a brief (< 5 words) version of the title>   |
++-----------------+--------------------------------------------------------+
+| Title           | <OEP title>                                            |
++-----------------+--------------------------------------------------------+
+| Last Modified   | <date string, in YYYY-MM-DD format>                    |
++-----------------+--------------------------------------------------------+
+| Author          | <list of authors' real names and                       |
+|                 | optionally, email addresses>                           |
++-----------------+--------------------------------------------------------+
+| Arbiter         | <Arbiter's real name and email address>                |
++-----------------+--------------------------------------------------------+
+| Status          | <Draft | Under Review | Deferred | Accepted |          |
+|                 | Rejected | Withdrawn | Final | Replaced>               |
++-----------------+--------------------------------------------------------+
+| Type            | <Architecture | Best Practice | Process>               |
++-----------------+--------------------------------------------------------+
+| Created         | <date created on, in YYYY-MM-DD format>                |
++-----------------+--------------------------------------------------------+
+| `Review Period` | <start - target end dates for review>                  |
++-----------------+--------------------------------------------------------+
+| `Resolution`    | <links to any discussions where the final              |
+|                 | status was decided>                                    |
++-----------------+--------------------------------------------------------+
+| `Replaces`      | <OEP number>                                           |
++-----------------+--------------------------------------------------------+
+| `Replaced-By`   | <OEP number>                                           |
++-----------------+--------------------------------------------------------+
+| `References`    | <links to any other relevant discussions               |
+|                 | or relevant related materials>                         |
++-----------------+--------------------------------------------------------+
 
 Abstract
 ========
 
-The abstract is a short (~200 word) description of the technical issue that
+The abstract is a short description of the technical issue that
```

Comment:
Why remove the length here?  It seems like that is a reasonable guide so people don't have to infer what `short` means to them.

Reply:
Overall, I reviewed OEP-1 through the lens of trying to simplify the process to encourage more folks to write down their decisions and have their designs reviewed before implementation.  (Given we have had only 13 OEPs written since its inception 2 years ago.)  So anything that I thought could be an unnecessary barrier, I tried to rephrase or remove.

Given that, having an explicit word count could be viewed as either a clear constraint to follow or an unnecessary restriction that the arbiter and reviewers can objectively comment on.  Right now, I'm erring on the side of less process - thinking we can always add process back in if needed.

Reply:
I agree with @feanil on this, though we could soften the wording to something like "We have found ~200 words to be a reasonable length for an abstract, but recognize that the length will vary based on the complexity of the OEP." to clarify both what we're looking for in length, and that it won't be a barrier to OEP approval.

Reply:
Sounds like a good compromise.  I'll wait a bit to hear if there are any other thoughts on this before making the update.

Reply:
**Guess what?**  Here's data about the number of words in the abstracts of the current 13 OEPs:

- **average**: 76 words
- **median**: 39
- **min**: 11 words
- **max**: 217

_I had happened to look at this as I began to write a new OEP today._

**What does this mean?**

1. Arbiters and reviewers have been satisfied with abstracts that have significantly fewer words than 200.
2. Even if there's a word-count requirement, people are using their own judgment.

**Given this,**
I'd rather keep OEP-1 as minimal as necessary.  And it doesn't seem like requiring or even suggesting 200 words for the abstract is necessary.  Different OEPs will tend to have different length of content and density of information in each of the sections.  I like how the current format helps people break it down into a consistent structure.  But then, we can leave it to the authors, arbiters, and reviewers how they choose to relay their information and decisions.